dora/core/common/src/test/java/alluxio/UfsUrlTest.java

```diff
@@ -35,13 +43,17 @@ public class UfsUrlTest {
   /**
    * Tests the {@link UfsUrl#createInstance(String)} constructor for basic Alluxio paths.
    */
+
+  @Rule
+  public final ExpectedException mException = ExpectedException.none();
```

Comment:
use `assertThrows` instead. it allows you to limit the scope of the assertion to a single method call.